scripts/globals/mobskills/flame_blast.lua

```diff
@@ -0,0 +1,29 @@
+---------------------------------------------
+--  Flame Blast
+--
+--  Description: Deals fire damage to enemies within a wide area of effect.
+--  Type: Magical
+--  Utsusemi/Blink absorb: Ignores shadows
+--  Range: 30'
+--  Notes: Used only by Wyrm in BCNM "Early Bird Catches the Wyrm"
+---------------------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/monstertpmoves")
+---------------------------------------------
+
+function onMobSkillCheck(target, mob, skill)
+    -- Only while flying
+    if mob:AnimationSub() ~= 1 then
```

Comment:
I noticed you have the same check in both versions, where as the other wyrms blast moves have the check in only 1 version. I don't know if that is a problem or not honestly,

Reply:
Hmm. Now that you say it, it is actually redundant since I created two dedicated sets for flying mode. One for the regular attack and one for TP moves. Since these are explicitly called for flying mode and then changed back, there's actually no real reason to check again. The other flying wyrms don't have a dedicated flying-mode TP set so they need to check for AnimationSub directly in the TP move skills. Whether or not that's good practice and whether or not there's maybe more flying TP moves from what we've seen, could be looked into in another issue.

I removed these checks completely and made it clear in the header comment that those skills should only be used for flying mode skill sets.

Good thing you came across this cause during this testing I also noticed a leftover of flame_blast in the ground skill set, which was removed, and more importantly:

I earlier resolved a merge conflict ( https://github.com/project-topaz/topaz/commit/0bd3bad899d48dfb4d2785d331bbc36d073f707f ) that was pushed in the meanwhile with changes to the blocking condition of being behind, to not being in front. I accepted these changes. But during this testing session today, this did **not** work at all. The skill won't get called anymore. I first observed this, then did some more testing and after just sitting there and counting 0/20 TP uses of dragon breath, I think it is safe to say this does not work.
I reverted this and it did work again. This might be considered a regression. Was this not tested?

I reverted this change only for Dragon breath cause it affects this PR. There's a lot of breaths that had that change made though and should be tested.